docs/ConnectWallet.md

```diff
@@ -1,30 +1,61 @@
 
-1. Allows connection to a full node
-2. Allows custom [gap limit](FAQ/FAQ-Wallet.md)
+:::tip
+By using an existing wallet, you can receive payments to the external wallet, without BTCPay Server knowing the wallet's private key. If a malicious attacker hacked your server and obtained the xpub, they could observe your transaction history, but cannot access the funds.
+:::
 
-## Recommended wallets
+Externally generated wallet can be used as:
+- [Existing hardware wallet](#use-an-existing-hardware-wallet)
+- [Existing software wallet](#use-an-existing-software-wallet)
 
-This page lists the recommended wallets for BTCPay Server. Click on the link and you will be redirected to a step by step tutorial on how to set up particular wallet with BTCPay Server.
+### Use an existing hardware wallet
 
-### Internal BTCPay Wallet
+Hardware wallets provide a good balance between security and ease of use. If you already have a hardware wallet set up, you can easily use it with your BTCPay Server. Thanks to built-in [hardware wallet integration](Vault.md) the xpub key from the hardware wallet is automatically added to your BTCPay Server. The integration further allows you to spend funds received to your store within BTCPay's [internal wallet](Wallet.md)
 
-- [Ledger Wallet](LedgerWallet.md)
-- [Coldcard Wallet]( 	ColdCardWallet.md)
+:::tip
+If you own a hardware wallet, follow the instructions on how to [use a an existing hardware wallet with your BTCPay Server](Vault.md)
+:::
 
-### External Wallets
+### Use an existing software wallet
+
+Using an existing software wallet assumes that you already have an external wallet created and backed up. In theory, any mobile/desktop wallet that provides an extended public key should work, however, most wallets have technical limitations [(gap-limit)](FAQ/FAQ-Wallet/#missing-payments-in-my-software-or-hardware-wallet) that may cause serious user-experience problems for you later on.
+
+For that reason, we recommend that you only use software wallets listed below.
 
-- [Wasabi Wallet](WasabiWallet.md)
 - [Electrum Wallet](ElectrumWallet.md)
+- [Wasabi Wallet](WasabiWallet.md)
+
+Click on the link above you will be redirected to a step by step tutorial on how to set up particular software wallet with BTCPay Server.
+
+To spend and manage the funds received to your external software wallet, you can use an [internal BTCPay Wallet](Wallet.md) and sign a transaction with your private key or simply manage the funds in that external wallet itself.
+
+## Create a new wallet
+
+If you do not have an existing wallet, you can generate it within your BTCPay Server. If are using a [third-party host](ThirdPartyHosting.md), this option needs to be explicitly enabled by the server admin. Generating a new wallet in an environment you don't trust is discouraged.
```

Comment:
hmmm not sure how to word this but I think it would be better to convey the message that: if you do not know, you should consider them untrustworthy... since lots of people are thinking ThirdParty hosts are safe since they are in btcpay docs but that does not mean it's true, we do not know.